pkg/restic/command.go

```diff
@@ -35,7 +35,7 @@ type Command struct {
 
 // StringSlice returns the command as a slice of strings.
 func (c *Command) StringSlice() []string {
-	res := []string{"/restic"}
+	res := []string{"restic"}
```

Comment:
Should we do `exec.LookPath` somewhere?

Reply:
why would we need that? that would get us the full path, but if it's in $PATH then we don't need it, right?

Reply:
I couldn't remember if it would find it or not. If it works as-is, no need.